src/crdt.ts

```diff
@@ -55,6 +59,11 @@ export class CRDT<T extends DocTypes> {
   readonly logger: Logger;
   readonly sthis: SuperThis;
 
+  // Subscriptions
+  _listening = false;
+  readonly _listeners = new Set<ListenerFn<T>>();
```

Comment:
i'm not a fan on use Set of functions --- please use Map<string, Fn> and generate string this sthis.nextid()